src/_help/publish-documentation/deploy-and-release-management.md

```diff
@@ -0,0 +1,125 @@
+
+### Deploy with CI
+
+### Deploy using the GitHub Action
+
+Our GitHub Action allows you to easily integrate Bump.sh into your projects by simply adding a workflow file.
+
+If you want to deploy your documentation after each push, you can use the following file: `.github/workflows/bump-deploy.yml`
+
+```
+name: Deploy documentation
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy-doc:
+    name: Deploy API doc on Bump.sh
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Deploy API documentation
+        uses: bump-sh/github-action@v1
+        with:
+          doc: <BUMP_DOC_ID>
+          token: ${{secrets.BUMP_TOKEN}}
+          file: doc/api-documentation.yml
+```
+
+## Release management
+
+Publishing a new version of your documentation typically follows a two-step process:
+- You upload a new version of your documentation through a specification file.
+- This new version is validated and deployed, becoming the "visible" version of your documentation. The changelog will also be update, reflecting changes between the current and previous version.
+
+By default, these two steps are automatically linked: uploading a new version results in its publication. However, it is possible to dissociate them using the Manual Release mode.
+
+The Deployments section of your documentation provides access to the history of all versions uploaded on Bump.sh, including their status (`enqueued`, `deploying`, `deployed`, or `errored`) and their release status (`Released` or `Not released`).
+
+![](/images/help/deployments-list.png)
+
+### Manual Release
+
+The Manual Release mode was recently introduced on Bump.sh and is still in beta. Please don't hesitate to contact us if you have any questions.
+
+The Manual Release mode is an option that allows you to separate the upload step from the publication step of your API documentation.
+
+![](/images/help/manual-release-toggle.png)
+
+When activated, uploading a version of your API documentation will no longer automatically publish it. We continue to prepare this version on our side (and you can view a preview and its changelog), but the release step becomes manual. This means you can decide when to make a version live for your API users.
+
+This option allows you to continue working on your documentation without worrying about the impact on your users. For example, you can prepare a future version of your API and its documentation with members of your organization and launch its publication when the time is right.
+
+#### Release manually a deployment
+
+From the list of your deployments, select a version of your documentation with the "unreleased" status.
+
+![](/images/help/deployments-list-not-released.png)
+
+The deployment preparation page will appear, allowing you to customize it by adding a title or a brief description, for example, to accompany this new version. These fields support Markdown to make the result more readable.
```

Comment:
Add a link to our Markdown support.
It's more about adding context than customizing.